### PR TITLE
address CVE warnings

### PR DESCRIPTION
**Pull request by pgwillia**
While these are all security updates on the whole they don't represent vectors that would affect us.

- bump omniauth from 1.9.0 to 1.9.1 [PR#1513](https://github.com/ualbertalib/jupiter/pull/1513) -- already addressed
- bump puma from 4.3.1 to 4.3.3 [CVE#2020-5247](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/puma/CVE-2020-5247.yml) -- not used in production
- bump loofah from 2.3.0 to 2.4.0 [CVE#2019-15587](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/loofah/CVE-2019-15587.yml) -- no republished SVG elements
- bump nokogiri from 1.10.4 to 1.10.9 [CVE#2019-13117](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/nokogiri/CVE-2019-13117.yml)
- bump brakeman from 4.7.0 to 4.8.0 [CVE#2019-18409](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/brakeman/CVE-2019-18409.yml) -- local user vector

#1515

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -576,4 +576,4 @@ DEPENDENCIES
   wicked
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```
Comment (pgwillia) [inline Gemfile.lock]: I can't remember if this is an issue any more or not. May I should revert this part.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,16 @@ and releases in Jupiter project adheres to [Semantic Versioning](http://semver.o
...
 
+## [1.2.20] - 2020-03-03
+
+### Security
+- bump omniauth from 1.9.0 to 1.9.1 [PR#1513](https://github.com/ualbertalib/jupiter/pull/1513)
+- bump puma from 4.3.1 to 4.3.3 [CVE#2020-5247](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/puma/CVE-2020-5247.yml)
+- bump loofah from 2.3.0 to 2.4.0 [CVE#2019-15587](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/loofah/CVE-2019-15587.yml)
+- bump nokogiri from 1.10.4 to 1.10.9 [CVE#2019-13117](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/nokogiri/CVE-2019-13117.yml)
+- bump brakeman from 4.7.0 to 4.8.0 [CVE#2019-18409](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/brakeman/CVE-2019-18409.yml)
+- add `noopener noreferrer` when opening a link in a new tab
```
Comment (pgwillia) [inline CHANGELOG.md]: brakeman found this so taking the opportunity to improve the code.

File: Gemfile.lock (hunk 1 above)
Comment (mbarnett) [reply Gemfile.lock]: I was under the impression it was still an issue? Might be best to confirm with Henry? There's no rush on most of these, as we're not impacted by most of them

Comment (pgwillia) [reply Gemfile.lock]: Henry checked.  It is.
```
# bundler --version
Bundler version 1.17.3
```

**Pull request merged**